.github/workflows/ci.yml

```
@@ -32,15 +45,31 @@
+          echo "Elasticsearch 실행을 위해 준비중..."
+          until curl -s http://localhost:9200/_cluster/health | grep -q '"status":"green\|yellow"'; do
+            echo "Elasticsearch 실행 대기중..."
+            sleep 10
+          done
+          echo "Elasticsearch 준비 완료"
+
+      - name: Nori plugin 설치 및 Elasticsearch 재시작
+        run: |
+          docker exec $(docker ps -q --filter ancestor=docker.elastic.co/elasticsearch/elasticsearch:9.0.3) \
+          bin/elasticsearch-plugin install analysis-nori
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)